crates/rome_formatter/tests/specs/js/static_member.expected.js

```diff
@@ -0,0 +1,3 @@
+a.b;
+c?.d;
+e.#f
```

Comment:
Shouldn't there be a `;` too?
```suggestion
e.#f;
```

Reply:
Hah! I thought this was a bug somewhere, and then I printed the diagnostics and saw ”private identifiers are unsupported“. I'll remove this test for now.

Reply:
Should work fine after  #1851 landed